oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurer.java

```diff
@@ -80,6 +82,8 @@
 			OAuth2TokenRevocationEndpointFilter.DEFAULT_TOKEN_REVOCATION_ENDPOINT_URI, HttpMethod.POST.name());
 	private final RequestMatcher jwkSetEndpointMatcher = new AntPathRequestMatcher(
 			JwkSetEndpointFilter.DEFAULT_JWK_SET_ENDPOINT_URI, HttpMethod.GET.name());
+	private final RequestMatcher oidcProviderMetadataEndpointMatcher = new AntPathRequestMatcher(
+			"/.well-known/openid-configuration", HttpMethod.GET.name());
```

Comment:
Let's add "/.well-known/openid-configuration" as a constant in the `Filter`, same as all other `Filter`'s

Reply:
Done.

Most of the other filters endpoints will be configurable through ProviderSettings though. The defaults are also in the ProviderSettings (hardcoded as well). Is there something specific we want to do about this ?